ext/bcmath/libbcmath/src/bcmath.h

```diff
@@ -55,7 +55,8 @@ typedef struct bc_struct
 #include "config.h"
 #endif
 
-#include "php.h"
+#include <stdbool.h>
+#include "php.h" /* Needed for safe_pemalloc() in init.c */
```

Comment:
Was this added accidentally?

Reply:
Not really, as I was tweaking some other stuff at the same time, but can split it out.

Reply:
Oh, I missed this added a comment and is thus shown as new. It's fine, no need to split then.